src/tests/roc_core/test_parse_units.cpp

```diff
@@ -53,5 +53,37 @@ TEST(parse_duration, parse) {
     CHECK(result == 123 * Hour);
 }
 
+TEST_GROUP(parse_size) {};
```

Comment:
nit: We usually use one test group per file, e.g. here it would be "parse_units".